components/use-cases/datahub/datahub.md

```diff
@@ -0,0 +1,31 @@
+---
+title: DataHub
+permalink: /components/use-cases/datahub/
+---
+
+* TOC
+{:toc}
+
+The DataHub use case template provides a one-click setup of an end-to-end flow for populating Keboola metadata 
+into a DataHub data catalog. This includes extracting data from both the Telemetry and Metadata extractors 
+before transforming the data to calculate table properties and lineage. You can choose whether or not to include schemas, 
+properties, or tags from the DataHub writer.
```

Comment:
podobně jako u advertising-platforms mě tady chybí popis toho, co se vlastně dozvím - dobře, dostanu telemetry data a odpoví mi to na jaké otázky?

Reply:
@jordanrburger can you give us more info about the datahub template please?